### PR TITLE
[IPS-1319] Update docs to reflect the Allowed Web Origins wildcards support

### PR DESCRIPTION
**Pull request by asmclean**
Add similar verbiage to what is used for `Allowed Origins (CORS)`.

The majority of these updates have been made in Contentful:

- Application Settings: https://auth0-docs-staging.herokuapp.com/docs/contentful/7wT5jc2JhV8eABLmTN4Dhe

- Wildcards for Subdomains: https://auth0-docs-staging.herokuapp.com/docs/contentful/3s1GOgcQuAo9E22ri7c465

- Cross-Origin Authentication: https://auth0-docs-staging.herokuapp.com/docs/contentful/5lHChXX5fVlZxo4U06zcdp

I didn’t include all the links to the “query string” changes. Finding and fixing those in various docs. The changes to the architecture scenarios docs will need to be made separately because we haven’t migrated those yet. I’ll take care of those separately. 

- Tami Goodall

### CODE REVIEW

File: articles/dashboard/reference/settings-application.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ You can provide up to 100 URLs in the **Allowed Callback URLs**, **Allowed Web O
...
 
-- **Allowed Web Origins**: List of URLs from where an authorization request using [`web_message` as the response mode](/protocols/oauth2#how-response-mode-works) can originate from. You can specify multiple valid URLs by comma-separating them. For production environments, verify that the URLs do not point to localhost.
+- **Allowed Web Origins**: List of URLs from where an authorization request using [Cross-Origin Authentication](/cross-origin-authentication), [Device Flow](/flows/guides/device-auth), and [`web_message` as the response mode](/protocols/oauth2#how-response-mode-works) can originate. You can specify multiple valid URLs by comma-separating them. For production environments, verify that the URLs do not point to localhost. You can use the star symbol as a [wildcard for subdomains](/applications/reference/wildcard-subdomains) (`*.google.com`). Paths, querystrings, and hash information are not taken into account when validating these URLs (and may, in fact, cause the match to fail).
```
Comment (luuuis) [inline articles/dashboard/reference/settings-application.md]: I suggest to replace `querystring` (not a word) with `query string` in this paragraph and others in the file.

File: articles/applications/reference/wildcard-subdomains.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ You can use wildcards for subdomain URL registration in your application configu
 * **Allowed Callback URLs**: Set of URLs to which Auth0 is allowed to redirect users after they authenticate.
 * **Allowed Logout URLs**: List of URLs to which you can redirect users after they log out from Auth0.
 * **Allowed Origins (CORS)**: Set of URLs that will be allowed to make requests from JavaScript to Auth0 API (typically used with CORS).
+* **Allowed Web Origins**: List of URLs from where an authorization request using [`web_message` as the response mode](/protocols/oauth2#how-response-mode-works) can originate.
```
Comment (luuuis) [inline articles/applications/reference/wildcard-subdomains.md]: How about the other usage sites as listed below (device flow etc)?

**Review by luuuis — DISMISSED** [review]

File: articles/dashboard/reference/settings-application.md (hunk 1 above)
Comment (asmclean) [reply articles/dashboard/reference/settings-application.md]: https://github.com/auth0/docs/pull/9275/commits/306ea260104cd4de946e8aa3ff50f4dfa4a0fe2c

File: articles/applications/reference/wildcard-subdomains.md (hunk 2 above)
Comment (asmclean) [reply articles/applications/reference/wildcard-subdomains.md]: https://github.com/auth0/docs/pull/9275/commits/306ea260104cd4de946e8aa3ff50f4dfa4a0fe2c

**Review by luuuis — DISMISSED** [review]

**Comment by tamigoodall** [comment]
Hi @asmclean - We are in a content freeze in preparation for our content move to a new Contentful CMS. We will get the update made as soon as possible after this week.

**Review by ziluvatar — DISMISSED** [review]

**Review by luuuis — APPROVED** [review]

**Comment by rcocetta** [comment]
Hello, 
Is there any update on when this PR can be merged?

**Comment by tamigoodall** [comment]
Updates made in Contentful and separate PRs for redirects changes and `query string` changes in content not in Contentful.

**Pull request closed without merging**